docs/useNotify.md

```diff
@@ -47,6 +37,21 @@ notify('item.created', { type: 'info', messageArgs: { resource: 'post' } });
 notify('Element updated', { type: 'info', undoable: true });
 ```
 
+## Parameters
+
+The hook takes no argument and returns a callback. The callback takes 2 arguments:
```

Comment:
It's weird to mention 2 arguments and to illustrate it with a table with more than 2 rows. I'd convert the description of the options to a list.